carshare/settings.py

```diff
@@ -265,3 +266,9 @@ def env_get_list(variable, default):
         },
     },
 }
+
+# contact details
+CONTACT_PHONE = '01720 575 301'
+CONTACT_CARSHARE_EMAIL = 'carshare@ioscv.co.uk'
+CONTACT_EMAIL = 'info@ioscv.co.uk'
```

Comment:
This one doesn't look like it's used anywhere in the PR? If it's not needed, can we just call the other one `CONTACT_EMAIL` and do away with this one?